examples/scripts/sentiment_tuning.py

```diff
@@ -45,7 +45,6 @@ class ScriptArguments:
         default=1, metadata={"help": "the number of gradient accumulation steps"}
     )
     early_stopping: Optional[bool] = field(default=False, metadata={"help": "whether to early stop"})
-    target_kl: Optional[float] = field(default=6, metadata={"help": "kl target for early stopping"})
```

Comment:
This field seems to have been removed by mistake?

Reply:
Hi Younes,

You will find that `target_kl` already exists on L57 with a much smaller value.

I dug deeper and found that `PPOConfig` has two configs `target` and `target_kl`, where `target` has a default value of 6. So I assume the first duplicate `target_kl` config here was meant to be `target`. However, `target` is NOT used to populate PPOConfig at L64, so I just removed it.

Regards,

Felix

Reply:
great point, thank you !

Reply:
I think this is actually a bug from here: https://github.com/lvwerra/trl/commit/1620da371a77e0c7030ff3093fa0d7814de575a0
we overloaded the `target_kl` term - we should rename it!

Reply:
cc @edbeeching

Reply:
@lvwerra as much as I love introducing bugs into trl. I think this time it was @younesbelkada , in the **Big refactor of examples and documentation** (https://github.com/lvwerra/trl/pull/509). [Here](https://github.com/lvwerra/trl/commit/5c7bfbc8d9aeabee893290cc02121d7260636978#diff-ecb04b8f3ca12c6ad0d07746a2f836b9cacbe468fe7fb148b4148fb77af5a73e)

I agree to rename to `early_stop_kl`, or something